heartbeat/CTDB.in

```diff
@@ -1,4 +1,4 @@
-#!/bin/sh
+#!@BASH_SHELL@
```

Comment:
Just a question: is this a common pattern these days? I was used to `/bin/bash` or `/usr/bin/env bash`.